### ISSUE TITLE
How long does one epoch take?

### ISSUE THREAD

**Issue by cissoidx** [Alibaba-MIIL/ImageNet21K#31]
I am pretraining from scratch the processed winter version of imagenet21k, 8*V100. 3 hours passed and Epoch 0 has not yet finished. It seems the training has crashed somehow. In the figure, it shows the GPU stats using wandb. I guess this is not normal, right? What would you suggest to change?
<img width="1424" alt="截屏2021-07-10 下午5 58 52" src="https://user-images.githubusercontent.com/17111388/125159198-865e1480-e1a8-11eb-9951-b8653a72c27a.png">

**Comment by cissoidx** [comment]
<img width="456" alt="截屏2021-07-10 下午8 14 00" src="https://user-images.githubusercontent.com/17111388/125162605-671cb280-e1bb-11eb-8b84-398cf8a57ad9.png">
Ran it again and similar crash happened. I monitored loss per 1000 batches, and it seems the losses in the 8 GPUs are normal. The losses are not exploding anyway, although they fluctuate more and more violent until it crashes.

**Comment by mrT23** [comment]
a general tip for the future - when an article gives you a training code, run it first and see what happens, before making changes.

run the original training code first. an epoch shouldn't take 3 hours on 8xV100

**Comment by SJLeo** [comment]
could you provide the training parameter configuration when using 8*V100

**Comment by cissoidx** [comment]
i am using the default parameters.
I am closing this because I fould out that the issue depicted in the figures above is due to IO problem of the system.
thanks for your replys.
